### PR TITLE
Update documentation of read_csv/tsc/eurostat (Issue 162)

### PR DESCRIPTION
**Pull request by alixdamman**

### CODE REVIEW

File: larray/core.py [hunk 1]
```diff
@@ -9265,8 +9265,8 @@ def read_csv(filepath, nb_index=None, index_col=None, sep=',', headersep=None, n
 
     Parameters
     ----------
-    filepath : str
-        Path where the csv file has to be read.
+    filepath : str or any object with a read() method
+        Path where the csv file has to be read or a file handle or a StringIO object.
```
Comment (gdementen) [inline larray/core.py]: unsure we should explicitly mention StringIO, because that is just one kind of file handle, right?

File: larray/core.py [hunk 2]
```diff
@@ -9265,8 +9265,8 @@ def read_csv(filepath, nb_index=None, index_col=None, sep=',', headersep=None, n
 
     Parameters
     ----------
-    filepath : str
-        Path where the csv file has to be read.
+    filepath : str or any object with a read() method
```
Comment (gdementen) [inline larray/core.py]: "any file-like object" instead of "any object with a read() method"???

File: larray/core.py [hunk 3]
```diff
@@ -9358,44 +9358,44 @@ def read_csv(filepath, nb_index=None, index_col=None, sep=',', headersep=None, n
...
 
-def read_hdf(filepath, key, na=np.nan, sort_rows=False, sort_columns=False, **kwargs):
+def read_hdf(path_or_buf, key, na=np.nan, sort_rows=False, sort_columns=False, **kwargs):
     """Reads an array named key from a HDF5 file in filepath (path+name)
 
     Parameters
     ----------
-    filepath : str
-        Filepath and name where the HDF5 file is stored.
+    path_or_buf : str or buffer
```
Comment (gdementen) [inline larray/core.py]: the argument name should be coherent. Either filepath_or_buffer everywhere or path_or_buf everywhere (or path_or_buffer). I do not really care either way.

File: larray/core.py (hunk 2 above)
Comment (alixdamman) [inline larray/core.py]: Taken from doc of pandas.read_csv but I agree with you. I'll change that.

**Comment by alixdamman** [comment]
update

File: larray/core.py (hunk 2 above)
Comment (gdementen) [inline larray/core.py]: but you did not?

Comment (alixdamman) [inline larray/core.py]: Now it should be OK.

**Pull request merged**